server/charts/historian/templates/gitrest-configmap.yaml

```diff
@@ -31,6 +31,9 @@ data:
             "lib": {
                 "name": "{{ .Values.gitrest.git.lib.name }}"
             },
+            "filesystem": {
+                "name": "{{ .Values.gitrest.git.filesystem.name }}"
```

Comment:
I thought the idea was that we'll use RedisFS for some docs, and a different FS (e.g. NodeFS) for other docs depending on use-case. Is that coming later? For example, if we only want "ephemeral doc" usecase on the Redis FS, then using this current config-based setup, we'd have to enable RedisFS on a full cluster, then direct all ephemeral-doc traffic to only the RedisFS clusters.